docs/automation-actions.md

```diff
@@ -490,3 +491,36 @@ has:
   jira_ticket_in_title: {{ pr.title | includes(regex=r/\b[A-Za-z]+-\d+\b/) }}
   jira_ticket_in_desc: {{ pr.description | includes(regex=r/atlassian.net\/browse\/\w{1,}-\d{3,4}/) }}
 ```
+
+
+#### `update-title` :fontawesome-brands-github:
+This action, when triggered, updates the PR title with new content.
+
+This is a managed action. When a PR updates, the existing comments that were added by gitStream are re-evaluated, and those that are not applicable are removed.
+
+<div class="filter-details" markdown=1>
+
+| Args       | Usage | Type      | Description                              |
+| -----------|-------|-----------|----------------------------------------- |
+| `title` | Required | String     | Sets the PR title |
+| `concat_mode` | Optional | String     | `replace` by default, the mode to concatenate the new description with the existing one. Possible values: `prepend`, `append`, `replace` |
+</div>
+
+For example, this automation updates the PR title with the ticket info if present in the PR title.
+
+```yaml+jinja title="example"
+automations:
+  add_jira_to_title:
+    if:
+      - {{ has.jira_ticket_in_desc and (not has.jira_ticket_in_title) }}
+    run:
+      - action: update-title@v1
+        args:
+          concat_mode: prepend
+          title: |
+            {{ pr.description | capture(regex=r/\b[A-Za-z]+-\d+\b/) }} -
```

Comment:
it took me time to understand (until Shaked explain to me) what you mean here because I know that you want to put a title, but instead, you use regex....
What can help to understand it, put variable : `jira_ticket_number_from_description: pr.description | capture(regex=r/\b[A-Za-z]+-\d+\b/)` and then use this variable...